.github/workflows/publish.yml

```diff
@@ -7,13 +7,16 @@ on:
 
 permissions:
   contents: read
-  packages: write
+  checks: write
```

Comment:
We don't publish to our internal package store anymore, but we do want to publish test output (as a check)